File: nodejs/models/VerificationModel.js
/**
* Modèle pour la vérification de compte
* Représente le modèle de données reçu lors de l'envoi d'une requete LPL de vérification de compte partenaire
* @author  Bastien Caubet <bastien@nextinpact.com>, Luc Raymond <luc@nextinpact.com>
*/ 
module.exports = VerificationModel;

function VerificationModel(obj) {
    this.Password = obj.Password;
    this.Mail = obj.Mail;
    this.CodeUtilisateur = obj.CodeUtilisateur;
}

File: asp.net/sdk-lpl-mvc5/sdk-lpl-mvc5/Models/VerificationModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace sdk_lpl_mvc5.Models
{
    /// <summary>
    /// Modèle pour la vérification de compte
    /// Représente le modèle de données reçu lors de l'envoi d'une requete LPL de vérification de compte partenaire
    /// </summary>
    public class VerificationModel
    {
        public string CodeUtilisateur { get; set; }
        public string Mail { get; set; }
        public string Password { get; set; }
    }
}
